backend/ingredient_endpoints.py

```diff
@@ -9,10 +9,12 @@
 from botocore.exceptions import ClientError
 from .models import Ingredient, SessionLocal
 from .schemas import IngredientCreate, IngredientUpdate, IngredientResponse
+from dotenv import load_dotenv
 
 router = APIRouter()
 logger = logging.getLogger(__name__)
 
+load_dotenv()
```

Comment:
@alicekalemba  Added this to allow the .env file to be loaded. it was working on your computer because you were already logged in to AWS. it wasnt working on mine. - issue is it wasnt reading the values from the .env file